docs/faq.md

```diff
@@ -8,42 +67,95 @@ be charged for anything with the exception of the following actions:
 
-    Hi DNAnexus,
+### Can I get a Microsoft Word version of the DAA?
+No. If we provide an editable format, we cannot ensure that the legal document has not been changed. Since we do not accept different versions of this agreement, we unfortunately cannot provide the document in Word format.
 
-      Would you please assist me in deleting my St. Jude Cloud account? My username is _____.
+### Where can I find the latest version of the Data Access Agreement (DAA)?
+We keep [our site](./guides/data/data-request.md) up to date with the latest version on the Data Access Agreement for you to download, or you can download a copy
+[here](https://platform.stjude.cloud/access_form).
 
-    Thank you!
+### Where do I submit the Data Access Agreement (DAA)?
+You can submit your Data Access Agreement in the drag and drop box on the [last step of the data request process](./guides/data/data-request.md#requesting-data).
 
-#### Where can I find the Terms of Service or the Privacy Policy?
+### What if I did not fill out the Data Download Permission section of the original DAA, but now I want to download data?
+Since this would be a change in terms from the original agreement, you would need to fill out a new DAA (including the [Data Download Permission section](./guides/forms/how-to-fill-out-DAA.md#data-download-permission) for any datasets you want to download.
 
-You can find the Terms of Service
-[here](https://stjude.cloud/terms-of-use.html) and the Privacy Policy
-[here](https://platform.stjude.cloud/privacy).
+
+### What clinical information is available about samples in St. Jude Cloud?
+Currently the only [clinical information](./guides/data/metadata.md#clinical-and-phenotypic-information) we provide is age at diagnosis, diagnosis, ethnicity, and sex. Unfortunately, even if we do collect other information, such as other supportive oncology data, it is not available on the cloud at this time. We are working towards being able to provide additional clinical annotations in the future.
+
+### Can I get a copy of IRB consent forms?
+Unfortunately, we will not be able to share blank consent forms at the current time. We have chosen to remain consistent with the requirements of the other major genomic data repositories in that (1) there is an internal vetting process by the St. Jude IRB to ensure samples may be shared with the research community, but (2) we do not share the informed consents with data requestors. It is important to remember that St. Jude Cloud is the platform upon which all St. Jude data is shared. This means that there are more than 100 consent forms + revisions for the various studies across St. Jude. Thus, there is an additional logistical barrier in that we simply don't have the bandwidth to pull together a packet containing all of this information for each requestor.
```

Comment:
This is minor, but change "requestors" to "requesters" to be consistent, since we're already using "requesters" in line 86. Based on [this](https://english.stackexchange.com/questions/29254/whats-the-difference-between-requester-and-requestor), I think "requesters" is correct for the documentation.